### ISSUE TITLE
No Issue - Search Engines

### ISSUE THREAD

**Issue by Eazylikesundaymorning** [DanielSmith1239/HQTriviaMaster#76]
Just curious what combination you guys have had the most luck with.

**Comment by NavkamalBains** [comment]
Also curious. Should I remove other search engines such as google, bing, yahoo etc. and just have sites like wikipedia, oxford english dictionary, google books, and other sites like that?

**Comment by sublime40oz** [comment]
Which file do you edit to change the sources?

**Comment by DanielSmith1239** [comment]
And wikipedia

**Comment by jolbol1** [comment]
I only use Google and Wikipedia, I did multiple tests with different sites and learnt that whatever you do you cant really change the results from the sites, it will be whats being searched that matters most.

**Comment by SyncedCyberSoul** [comment]
Aye, Google works but in my testing, wikipedia is the worst possible one to have. Try google scholar. Also, instead of having "www" before it put *.(domain here). That way it searches ALL of google and ALL of google scholar.

**Comment by rkundrata** [comment]
Yeah, I run 6 bots, all of which are using different variations of websites to search and whether it searches entire web or not.

**Comment by mericausa** [comment]
How do you use a second search? When I enter the second one it changes the (google) search in my keychain and seems to revert to the last one entered. Can you use two or more searches at once?

**Comment by soup919** [comment]
Who has a good list of websites to use for good probability in your Custom Search Engine? 
And are you using "search only included sites" OR "search the entire web and emphasize included sites"?
I'm getting like half of the answers being correct. 
A lot of questions are getting 100% 0% 0% now which are sometimes correct.

**Comment by westearth** [comment]
How do you add Wikipedia as a search engine? I’ve been trying in different ways but my search doesn’t give me any Wikipedia result

**Comment by soup919** [comment]
Who has a good list of websites to use for good probability in your Custom Search Engine?
And are you using "search only included sites" OR "search the entire web and emphasize included sites"?
I'm getting like half of the answers being correct.
A lot of questions are getting 100% 0% 0% now which are sometimes correct.
